File: src/main/java/wsmt/bursa/Apple.java
package wsmt.bursa;

import java.io.Serializable;

/**
 * Created by Tudor on 09-Apr-17.
 */
public class Apple extends Actiune implements Serializable {
	public static final String APPLE = "Apple.INC";
	
	public Apple() {
		super(APPLE, 56.34);
	}
}

File: PHP/bursa/Apple.php
<?php
/**
 * Created by IntelliJ IDEA.
 * User: Tudor
 * Date: 09-Apr-17
 * Time: 8:12 PM
 */

namespace bursa;
require("Actiune.php");



class Apple extends Actiune
{
	static $APPLE = "APPLE";
    function __construct() {
        $this->pret = 56.34;
        $this->nume = Apple::$APPLE;
    }

}